mozilla_bitbar_devicepool/test_run_manager.py

```diff
@@ -58,8 +59,8 @@ def __init__(self, wait=60, delete_bitbar_tests=False):
                     # ConnectionErrors.
                     time.sleep(5)
                     break
-                except requests.exceptions.ConnectionError:
-                    logger.error('Failed to tests for project %s' % project_name,
+                except (requests.exceptions.ConnectionError, RequestResponseError) as e:
+                    logger.error('Failed to get tests for project %s (%s: %s).' % (project_name, e.__class__, e.message),
```

Comment:
I know it is frowned upon to catch everything but we don't want to let any exceptions kill us. What would you say to
```
except:
    logger.exception('Failed to get tests for project %s' % project_name)
```
and we can drop the from testdroid import RequestResponseError.

Reply:
Yeah, I think the concern is that it would mask a trivial coding error/bug like a dictionary index lookup failure. I feel like I've encountered those cases before. In this case, we'd detect queues aren't decreasing, look at logs, and fix... so not a huge deal.

Done.